rna/admin.py

```diff
@@ -98,7 +98,7 @@ def copy_releases(self, request, queryset):
 
     def set_to_public(self, request, queryset):
         """ Set one or several releases to public """
-        queryset.update(is_public=True)
+        queryset.update(is_public=True, modified=datetime.now())
```

Comment:
It's best these days to use `django.utils.timezone.now()` so that you get a TZ aware datetime if `USE_TZ` is `True`. But I see others in this file don't do that, so this is probably fine for now.